mon-pix/app/services/errors.js

```diff
@@ -0,0 +1,17 @@
+import Service  from '@ember/service';
+
+export default class Errors extends Service {
+  errors = [];
+
+  push(error) {
+    this.errors.push(error);
+  }
+
+  shift() {
+    return this.errors.shift();
+  }
+
+  hasErrors() {
```

Comment:
(Remarque mineure )
Ca me fait bizarre de lire this.errors.hasErrors();
` return this.hasFailed ? false : this.errors.hasErrors();`
Je propose
```suggestion
  isNotEmpty() {
```
Pour avoir ` return this.hasFailed ? false : this.errors.isNotEmpty();`